test/OrchardCore.Commerce.Tests.UI.Shortcuts/Controllers/OrderController.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Lombiq.HelpfulLibraries.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrchardCore.Commerce.Abstractions;
+using OrchardCore.Commerce.AddressDataType;
+using OrchardCore.Commerce.Constants;
+using OrchardCore.Commerce.Controllers;
+using OrchardCore.Commerce.Fields;
+using OrchardCore.Commerce.Models;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Utilities;
+using OrchardCore.Mvc.Core.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using static OrchardCore.Commerce.Constants.ContentTypes;
+
+namespace OrchardCore.Commerce.Tests.UI.Shortcuts.Controllers;
+
+[DevelopmentAndLocalhostOnly]
+public class OrderController : Controller
+{
+    private readonly Lazy<IPaymentService> _checkoutService;
+    private readonly Lazy<IShoppingCartPersistence> _shoppingCartPersistence;
+    private readonly Lazy<IContentManager> _contentManager;
+    private readonly Lazy<IStripePaymentService> _stripePaymentService;
+
+    public OrderController(
+        Lazy<IPaymentService> checkoutService,
```

Comment:
Why is this variable called `checkoutService` instead of `paymentService`?

Reply:
Renaming leftover... 😓